{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/redirects/admin.py

```diff
@@ -75,14 +75,14 @@ def get_list_display(self, request):
 
         return ('old_path', 'new_path', 'type', 'test_redirect')
 
-    def get_form(self, request, obj=None, **kwargs):
+    def get_form(self, request, obj=None, change=False, **kwargs):
         kwargs["form"] = RedirectModelForm
 
         if not getattr(settings, 'REDIRECTS_ENABLE_REGEX', False):
             kwargs.update({
                 'exclude': ['regular_expression', 'test_path']
             })
-        form = super().get_form(request, obj, **kwargs)
+        form = super().get_form(request, obj=obj, change=change, **kwargs)
         return form
 
     def test_redirect(self, obj):
```

Comment:
need to `mark_safe` the return here or it renders the html:

`from django.utils.safestring import mark_safe`
`return mark_safe('<a target="_blank" href="{}">Test</a>'.format(escape(url)))`